tests/queries/0_stateless/00397_tsv_format_synonym.sql

```diff
@@ -9,3 +9,4 @@ SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT TSVWithN
 
 SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT TabSeparatedRaw;
 SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT TSVRaw;
+SELECT arrayJoin([1, 2, 3]) AS arr, 'hello' AS s1, 'world' AS s2 FORMAT Raw;
```

Comment:
Please add a tests with `RawWithNames` and `RawWithNamesAndNames`

Reply:
> Please add a tests with `RawWithNames` and `RawWithNamesAndNames`

Got it!